### PR TITLE
down with turbolinks

### PR DESCRIPTION
**Pull request by Sillson**
Removes turbolinks to fix g5_header issues and tagging

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title>G5NewsAndEventsService</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => false %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => false %>
```
Comment (johnlucia) [inline app/views/layouts/application.html.erb]: I think (but I'm not certain) that you can just totally remove these two lines.

**Comment by johnlucia** [comment]
👍

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (johnlucia) [inline app/views/layouts/application.html.erb]: actually, not the whole lines, just the turbolinks key/value pairs

**Comment by Sillson** [comment]
@johnlucia removed the turbolinks key-value for good measure.

**Pull request merged**